helpers/context.js

```diff
@@ -0,0 +1,14 @@
+require("express-async-errors");
+const jwt = require("jsonwebtoken");
+const { User } = require("../models/user");
+const validateToken = require("./validateToken");
+
+module.exports = async (req, res) => {
```

Comment:
Please, bare in mind to not declare a function as anonymous, but provide a naming for it.